Libs/vtkITK/vtkITKArchetypeImageSeriesReader.cxx

```diff
@@ -823,9 +826,76 @@ int vtkITKArchetypeImageSeriesReader::RequestInformation(
     this->Dictionary = itk::MetaDataDictionary();
     }
   ParseDictionary();
+
+  std::string range_keys[2] = {"min","max"};
+  std::string min_str;
+  std::string max_str;
+  if (itk::ExposeMetaData<std::string>(this->Dictionary, range_keys[0], min_str) &&
+      itk::ExposeMetaData<std::string>(this->Dictionary, range_keys[1], max_str))
+    {
+    double value = 0.0;
+    std::istringstream min_sstr(min_str);
+    while (min_sstr >> value)
+      {
+      this->MetaDataScalarRangeMinima.push_back(value);
+      }
+    std::istringstream max_sstr(max_str);
+    while (max_sstr >> value)
+      {
+      this->MetaDataScalarRangeMaxima.push_back(value);
+      }
+    }
+
```

Comment:
This is due to the different factories being tested in the following order:
* vtkITKArchetypeDiffusionTensorImageReaderFile
* vtkITKArchetypeImageSeriesVectorReaderSeries
* vtkITKArchetypeImageSeriesVectorReaderFile
* vtkITKArchetypeImageSeriesScalarReader

I'll check how much time this adds up, but my intuition is not that much.